forge/Dockerfile

```diff
@@ -41,4 +26,4 @@ ARG BEAM_VERSION=2.40.0
 # Install the same version of the package that we install the boot binary of
 RUN pip install --no-cache apache-beam[gcp]==${BEAM_VERSION}
 
-COPY --from=beamboot /beam-boot /usr/local/bin/boot
+COPY --from=apache/beam_python3.8_sdk:2.40.0 /opt/apache/beam/boot /opt/apache/beam/boot
```

Comment:
Does it matter if you pull from `python3.8_sdk` vs. `python3.9_sdk`?

Reply:
@cisaacstern i'll switch it to 3.9 but it shouldn't matter as I'm picking up just the built go binary. Should modify docs too.

Reply:
Yes, it does matter: the python versions need to be consistent all the way through, or else you get Dataflow runtime errors. There is an open JIRA ticket in the beam project to address this, but I'm having a hard time finding it now.

Reply:
@alxmrs I'm only copying the `boot` go binary, not the built python package. The python package is being installed with `pip` into our conda environment directly here. I don't want to bring in any python dependencies by copying out of the beam image.